.claude/skills/firebase-prod-campaigns.md

```
@@ -3,16 +3,40 @@
+# 1. Get user UID from email
+user_record = auth.get_user_by_email('$USER@gmail.com')
+uid = user_record.uid  # e.g., 'vnLp2G3m21PJL6kxcuAqmWSOtm73'
+
+# 2. Query nested path
+doc = db.collection('users').document(uid).collection('campaigns').document('CAMPAIGN_ID').get()
+```
+
+## Database Structure
 
-### Database Structure
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)